src/systems/renderer.js

```diff
@@ -48,6 +51,13 @@ module.exports.System = registerSystem('renderer', {
     }
   },
 
+  update: function () {
+    var data = this.data;
+    var sceneEl = this.el;
+    var renderer = sceneEl.renderer;
+    renderer.toneMappingExposure = data.exposure;
```

Comment:
I have no experience with exposure or tone mapping so this might be dumb. Are there use cases for different values of tone mapping and exposure? Should we always match them automatically?